### PR TITLE
Feature/upstream new mp spdz

### PR DESCRIPTION
**Pull request by kindlich**
Follow-up on (needs all those merged first, hence still a draft PR atm)
- carbynestack/carbynestack#37
- carbynestack/cli#19

Closes
- carbynestack/carbynestack.github.io#50

### CODE REVIEW

**Comment by kindlich** [comment]
Should be ready to review.
The general workflow (that the new tuples work and that the CLI works) were already tested here
https://github.com/carbynestack/carbynestack/pull/37#issuecomment-1123332940

Do we need another setup test, or is it fine with the above being done?

File: docs/getting-started/deployment.md [hunk 1]
```diff
@@ -168,9 +168,9 @@ time-consuming process, we provide pre-generated material.
 1. Download and decompress the archive containing the material using:
 
     ```shell
-    curl -O -L https://github.com/carbynestack/base-images/raw/3595c5427915b2f9e1f22804e3f742cda9e72312/fake-crypto-material.zip
-    unzip -d crypto-material fake-crypto-material.zip
-    rm fake-crypto-material.zip
+    curl -O -L https://github.com/carbynestack/carbynestack/raw/master/tuples/fake-crypto-material-0.2.zip
```
Comment (strieflin) [inline docs/getting-started/deployment.md:171]: Please replace `master` with a concrete commit sha to lower the risk of breaking the tutorial when updating `master`.

**Review by strieflin — CHANGES_REQUESTED** [review]
Will be ready for being merge after resolving one small finding.

File: docs/getting-started/deployment.md (hunk 1 above)
Comment (kindlich) [reply docs/getting-started/deployment.md]: Done

**Review by strieflin — APPROVED** [review]
LGTM

**Pull request merged**